docs/install_molgenis/guide-local-compile.md

```diff
@@ -75,10 +78,76 @@ By default MOLGENIS will try to connect to a node within cluster 'molgenis' on a
 elasticsearch.cluster.name=my-cluster-name
 elasticsearch.transport.addresses=127.1.2.3:9300,127.3.4.5:9301
 ```
-**Launching MOLGENIS**  
-Now that your database server, Elasticsearch and properties file have been configured, restart the Apache Tomcat server.
-If you open up a web browser and navigate to where your apache-tomcat applications are deployed (often this is localhost:8080) you should see the following:  
 
-![](../images//molgenis_home_logged_out.png?raw=true, "molgenis home page")  
+## Get the code
+
+create acount on github.com 
```

Comment:
If you want to run it, you only need to clone, not to fork. Is this perhaps a bit too high a barrier?